pkgs/development/compilers/chicken/5/update.sh

```diff
@@ -1,23 +1,18 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -i oil -p oil jq ocaml-ng.ocamlPackages_latest.sexp
+#! nix-shell -i oil -p oil chicken
+
 export URL_PREFIX="https://code.call-cc.org/egg-tarballs/5/"
 cd $(nix-prefetch-url \
      'https://code.call-cc.org/cgi-bin/gitweb.cgi?p=eggs-5-latest.git;a=snapshot;h=master;sf=tgz' \
      --name chicken-eggs-5-latest --unpack --print-path | tail -1)
-const eggExpr = '
-  def toarray: if type=="array" then . else [.] end;
-  if type=="array" then map({(first): .[1:]}) | add else . end |
-    {synopsis:     .synopsis | toarray | first | tostring,
-     version:      env.EGG_VERSION,
-     sha256:       env.EGG_SHA256,
-     dependencies: (.dependencies // []) | toarray | map(toarray) | map(first),
-     license:      .license | toarray | first | ascii_downcase | sub(" ";"-")}'
+echo $(pwd)
```

Comment:
A leftover from debugging?

Reply:
woops, should've checked the diff more thoroughly